pepper-apis/docs/specification/src/components/schemas.yml

```
@@ -2713,6 +2713,8 @@ MedicalProvider:
       type: string
     state:
       type: string
+    country:
+      type: string
```

If this is not a required field, it should be marked as `nullable`
```suggestion
    country:
      type: string
      nullable: true
```